client/components/common/ConfirmationModal.vue

```diff
@@ -42,8 +45,11 @@ export default {
     };
   },
   computed: {
-    info() {
-      return this.context.item.name;
+    publish() {
```

Comment:
There shouldn't be a notion of publishing within general confirmation modal. Please make sure it stays generic.